go/vt/vttablet/grpctmclient/client.go

```diff
@@ -88,10 +119,11 @@ func (client *Client) dial(tablet *topodatapb.Tablet) (*grpc.ClientConn, tabletm
 	if err != nil {
```

Comment:
Now that you've properly wired up a `context.Context` to this API, we can change the `grpcclient.Dial` call here to `grpcclient.DialContext`, which is going to fix an annoying issue, as seen here: https://github.com/vitessio/vitess/pull/8387#discussion_r659670417

Can you update this PR accordingly? That'll unblock the other PR. :)

Reply:
Happy to swap to DialContext, but per my comment on the other PR there's more work needed to remove that hack